docs/design/vm-metadata-01-design.md

```
@@ -0,0 +1,934 @@
+
+增加全局配置项 `vm.metadata.enabled`（Boolean，**默认为 false**），开启/关闭记录虚拟机元数据。
+
+**理由**：注册虚拟机仅用于有容灾需求的场景。对于 99.9% 普通用户来说不会用到此功能。
+
+---
+
+## 2. 核心 DTO 结构
+
+### 2.1 VmInstanceMetadataDTO
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)